standup_notes/__main__.py

```diff
@@ -59,12 +58,9 @@ def main():
     if arguments.copy:
         call_func_for_specified_day(copy_note, arguments)
 
-    if arguments.post:
-        matter_post()
-
 
 def call_func_for_specified_day(func, arguments):
-    """Calls the proper function """
+    """Functions calls to make sure proper day arguments are being passed to thei respective functions"""
```

Comment:
'Functions calls' seems a little oddly worded... Maybe just remove the first 3 words and start with 'Makes sure'